draft-ietf-quic-transport.md

```diff
@@ -2108,21 +2108,13 @@ genuine migrations.  Changing port number can cause a peer to reset its
 congestion state (see {{migration-cc}}), so the port SHOULD only be changed
 infrequently.
 
-Endpoints that use connection IDs with length greater than zero could have their
-activity correlated if their peers keep using the same destination connection ID
-after migration. Endpoints that receive packets with a previously unused
-Destination Connection ID SHOULD change to sending packets with a connection ID
-that has not been used on any other network path.  The goal here is to ensure
-that packets sent on different paths cannot be correlated. To fulfill this
-privacy requirement, endpoints that initiate migration and use connection IDs
-with length greater than zero SHOULD provide their peers with new connection IDs
-before migration.
-
-Caution:
-
-: If both endpoints change connection ID in response to seeing a change in
-  connection ID from their peer, then this can trigger an infinite sequence of
-  changes.
+Endpoints that supply connection IDs with length greater than zero could have
+their activity correlated if their peers keep using the same destination
+connection ID after migration.  To ensure that packets sent on different paths
+cannot be correlated, endpoints SHOULD provide with new connection IDs before
```

Comment:
```suggestion
cannot be correlated, endpoints SHOULD supply new connection IDs to their peers before the peers migrate.
```

And drop the rest of this paragraph, I don't think it adds anything.

Reply:
#2414 might be the better PR to review then.  I've removed the duplicated text.

Reply:
I like the text there better, I've left a comment as well. Do you want to close this PR off in favor of that one?

Reply:
This should close when that does.  Closing this might have side-effects.